src/js/app.controller.js

```diff
@@ -24,20 +24,12 @@
     $scope.fileSearch = null;
     $scope.allFiles = {};
 
-    // vm.create = () => {
-    //   let result = vm.indexer.test();
-    //   alert(result);
-    // };
-
     vm.create = () => {
```

Comment:
In Angular, vm is called ViewModel. vm is used in place of or to represent the binding scope. It also limits the use of the $scope variable. So, its a standard variable name and in my code I used it to represent the binding scope, though it's allowed to use $scope along with it. As I learn more, I will update on the best practices. Thanks.